modules/serverless-rn-1-30-0.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-30-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.30
+
+{ServerlessProductName} 1.30 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[IMPORTANT]
+====
+{ocp-product-title} 4.13 is based on {op-system-base-full} 9.2.  {op-system-base} 9.2 is yet to be submitted for Federal Information Processing Standards (FIPS) validation. Although Red Hat cannot commit to a specific timeframe, we expect to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Information on updates will be available in the link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards Knowledgebase article].
+====
+
+[id="new-features-1-30-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.9.
+* {ServerlessProductName} now uses Knative Eventing 1.9.
+* {ServerlessProductName} now uses Kourier 1.9.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.9.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.9.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+
+* {FunctionsProductName} nows supports {pipelines-title} versions 1.10 and 1.11. Older versions of {pipelines-title} are no longer compatible with {FunctionsProductName}.
+
+* Eventing integration with {SMProductName} is now available as a Technology Preview (TP) feature.
++
+The integration includes the following:
+** `PingSource`
+** `ApiServerSource`
+** Knative Source for Apache Kafka
+** Knative Broker for Apache Kafka
+** Knative Sink for Apache Kafka
+** `ContainerSource`
+** `SinkBinding`
+** `InMemoryChannel`
+** `KafkaChannel`
+** Channel-based Knative Broker
+
+* {FunctionsProductName} users now have the ability to override the `readiness` and `liveness` probe values in the `func.yaml` file for individual Quarkus functions.
++
+See "Functions development reference guide" for guidance on Quarkus, TypeScript, and Node.js functions.
+
+* Beginning {ServerlessProductName}, Kourier controller and gateway manifests have the following limits and requests by default:
+** requests:
+*** cpu: 200m
+*** memory: 200Mi
+** limits:
+*** cpu: 500m
+*** memory: 500Mi
++
+See the "Overriding Knative Serving system deployment configurations" section of {ServerlessProductName} documentation.
```

Comment:
Can this be a link?

Reply:
@ReToCode Unfortunately, it cannot, as we cannot link from a module. However, I added this into the "Additional resources" heading after the 1.30 release notes section. Thanks!